core/src/rack.c

```diff
@@ -70,18 +70,22 @@ int score_on_rack(LetterDistribution *letter_distribution, Rack *rack) {
 void set_rack_to_string(Rack *rack, const char *rack_string,
                         LetterDistribution *letter_distribution) {
   reset_rack(rack);
-  for (size_t i = 0; i < strlen(rack_string); i++) {
-    add_letter_to_rack(rack, human_readable_letter_to_machine_letter(
-                                 letter_distribution, rack_string[i]));
+
+  uint8_t mls[50]; // idk
```

Comment:
nit: this should probably use the RACK_SIZE constant in some way if I am reading this correctly, but isn't really necessary as long as it's big enough.

Reply:
will change